<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>authorized</class>
 <widget class="QDialog" name="authorized">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change password</string>
  </property>
  <widget class="QPushButton" name="change_password">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>196</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Change password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>132</y>
     <width>113</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Repeat password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="repeated_password">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>152</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="login">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>32</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>192</y>
     <width>109</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Old password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>72</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>12</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="old_password">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>212</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>92</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="debug_info">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>236</y>
     <width>253</width>
     <height>53</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
